client/src/parsec/functions.ts

```diff
@@ -331,3 +333,89 @@ export async function isValidDeviceName(name: string): Promise<boolean> {
 export async function isValidInvitationToken(token: string): Promise<boolean> {
   return await libparsec.validateInvitationToken(token);
 }
+
+export async function listUsers(skipRevoked = true): Promise<Result<Array<UserInfo>, ClientListUsersError>> {
+  const handle = getParsecHandle();
+
+  if (handle !== null && window.isDesktop()) {
+    const result = await libparsec.clientListUsers(handle, skipRevoked);
+    if (result.ok) {
+      result.value.map((item) => {
+        item.createdOn = DateTime.fromMillis(item.createdOn as any as number);
```

Comment:
This is done because the bindings returns a number, but typed as a luxon DateTime. It means the object can be used by the UI as a DateTime later on, we just have to force a conversion first.